workflow-core/src/commonMain/kotlin/com/squareup/workflow1/Worker.kt

```diff
@@ -247,10 +202,7 @@ public interface Worker<out OutputT> {
      * builder functions that have the same output type.
      */
     public inline fun <reified OutputT> fromNullable(
-      // This could be crossinline, but there's a coroutines bug that will cause the coroutine
-      // to immediately resume on suspension inside block when it is crossinline.
-      // See https://youtrack.jetbrains.com/issue/KT-31197.
-      noinline block: suspend () -> OutputT?
+      crossinline block: suspend () -> OutputT?
```

Comment:
Also changed this to `crossinline` as that issue was fixed a number of years ago.